geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/analytics/HistogramJob.scala

```diff
@@ -47,14 +48,18 @@ class HistogramJob(args: Args) extends GeoMesaBaseJob(args) {
   val groupBy    = args.nonStrictList(GROUP_BY)
   val uniqueBy   = args.nonStrictList(UNIQUE_BY)
   val transforms = Option(args.list(TRANSFORM_IN).toArray).filter(_.nonEmpty)
+  val writeToAccumulo  = args.boolean(WRITE_TO_ACCUMULO)
 
   val input = GeoMesaInputOptions(dsParams, feature, filter, transforms)
+  lazy val dsParamsAccOutput = dsParams.updated("tableName", s"${dsParams.get("tableName").get}_stats")
```

Comment:
not that it will happen, but it may be safer to do a getOrElse to avoid the null.